README.md

```diff
@@ -133,21 +133,46 @@ Authenticating using [token authentication](https://ably.com/documentation/core-
 final clientOptions = ably.ClientOptions()
 // ..clientId = _clientId // Optionally set the clientId
   ..autoConnect = false
-  ..authCallback = (TokenParams tokenParams) async {
+  // If a clientId was set in ClientOptions, it will be available in ably.TokenParams.
+  ..authCallback = (ably.TokenParams tokenParams) async {
     try {
-      // If a clientId was set in ClientOptions, it will be available in Ably.TokenParams.
-      final tokenRequestMap =
-      await createTokenRequest(tokenParams: tokenParams);
+      // Send the tokenParamsMap (Map<String, dynamic>) to your server, using it to create a TokenRequest.
+      final Map<String, dynamic> tokenRequestMap = await createTokenRequest(
+              tokenParams.toMap());
       return ably.TokenRequest.fromMap(tokenRequestMap);
     } catch (e) {
-      print("Something went wrong in the authCallback:");
-      print(e);
+      print("Something went wrong in the authCallback: ${e}");
+      rethrow;
     }
   };
 this._ablyClient = new ably.Realtime(options: clientOptions);
 await this._ablyClient.connect();
 ```
 
+An example of `createTokenRequest`'s implementation making a network request to your server could be:
+```dart
```

Comment:
Before _and after_ code fenced segments it is desirable to include a blank line (vertical white space) to enhance readability when viewing the markdown raw or editing it. That also conforms with other instances in this file and elsewhere across our repositories.

Reply:
Thanks, I've actually updated this section in a different PR (https://github.com/ably/ably-flutter/pull/250). To avoid merge conflicts, I added the whitespace in 4b1a922e9bc97af9e1dc81ea5b7f1d1da1c4c651